shared/render/render/XMLRenderer.test.tsx

```diff
@@ -35,6 +35,25 @@ describe('XMLRenderer', () => {
     });
 
     it('renders decision'); // TODO
+
+    it('renders conditional ops as ifs', () => {
+      expect(XMLRenderer.toTemplate('roleplay', {}, ['{{ gold == 0 }} Test'], 0).toString())
+        .toEqual('<roleplay data-line="0"><p if=" gold == 0 "> Test</p></roleplay>');
+    });
+
+    it('renders nonconditional ops as output', () => {  
```

Comment:
Could you add a few more test cases, just to make absolutely sure this'll work as expected? Few I can think of:
- ternary operator
- assigning something to a dictionary, ie `a = {b: 'c'}`
- accessing something from an array, ie `a = [1,2,3]` then later on outputting `a[1]`
- conditional involving a dictionary